### PR TITLE
Add more detail to errors returned from Commit()

### PR DESCRIPTION
**Pull request by nalind**
Add some context to errors returned from the Commit() method.

### CODE REVIEW

**Comment by nalind** [comment]
[(5) [buildah] Get buildah to support docker image format](https://trello.com/c/Bu0QQfjd/482-5-buildah-get-buildah-to-support-docker-image-format)

File: commit.go [hunk 1]
```diff
@@ -47,22 +48,22 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
...
-		return err
+		return fmt.Errorf("error copying layers and metadata: %v", err)
 	}
 	if len(options.AdditionalTags) > 0 {
 		switch dest.Transport().Name() {
 		case storage.Transport.Name():
 			img, err := storage.Transport.GetStoreImage(b.store, dest)
 			if err != nil {
-				return err
+				return fmt.Errorf("error locating just-written image: %v", err)
```
Comment (TomSweeneyRedHat) [inline commit.go]: Could you add the value from b.store to the message to let the user know what it was looking at in particular when there was an error?  Or do you think that wouldn't help the debugging process later on down the road?

Comment (nalind) [inline commit.go]: github.com/containers/image/transports.ImageName() should include that information, adding its value to the error message.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+  	328d5ad

**Pull request closed without merging**